src/fparser/common/sourceinfo.py

```diff
@@ -307,7 +308,11 @@ def get_source_info(file_candidate):
         # It is closed on completion so as to return it to the state it was
         # found in.
         #
-        with open(file_candidate, 'r') as file_object:
-            return get_source_info_str(file_object.read())
+        from fparser.common.utils import make_clean_tmpfile
```

Comment:
Please update the docstring of the parent routine to follow rst markup.

Reply:
OK (I think). File objects are different in py2 and py3.